RevenueCatUI/CustomerCenter/Data/CustomerCenterConfigTestData.swift

```diff
@@ -103,7 +103,8 @@ enum CustomerCenterConfigTestData {
                 "cancel": "Cancel",
                 "back": "Back"
             ]
-        )
+        ),
+        support: .init(email: "support@revenuecat.com")
```

Comment:
Nitpick but maybe change this to something else just in case? Like
```suggestion
        support: .init(email: "test-support@revenuecat.com")
```